config/settings.yml

```diff
@@ -27,6 +27,10 @@ feature_flags:
     local: true
     staging: true
     production: false
+  case_concluded_page:
```

Comment:
I think as this is part of the means journey this needs to be under the 'means_journey' flag rather than its own flag for the page 👍